AppDashboard/lib/app_dashboard_data.py

```diff
@@ -291,15 +291,14 @@ def update_head_node_ip(self):
       logging.exception(err)
       return None
 
-  def update_request_info(self, app_id):
+  def update_request_info(self, version_key):
     """ Queries the AppController to get request information for the given
-    application, storing it in the Datastore for later viewing.
+    version, storing it in the Datastore for later viewing.
 
     Args:
-      app_id: A string, the application identifier.
+      version_key: A string specifying the version key.
```

Comment:
would you mind mentioning that it's formatted appid_service_version_revision since someone working on the dashboard may not know the inner workings.